Patch origin: in-house
Patch status: Solaris-specific; not suitable for upstream

The daemon name is just ocspd on Solaris.

The description is outdated. It needs to reflect what we have on Solaris.

ocspd's configuration file is /etc/ocspd/ocspd.xml on Solaris.

Syntax error. =back is missing.

--- openca-ocspd-3.1.2/docs/ocspd.3.pod	2016-07-06 15:54:02.146059031 -0700
+++ openca-ocspd-3.1.2/docs/ocspd.3.pod	2016-07-14 13:33:06.330761183 -0700
@@ -3,11 +3,11 @@
 
 =head1 NAME
 
-	openca-ocspd - OCSP Daemon
+	ocspd - OCSP Daemon
 
 =head1 SYNOPSIS
 
-B<openca-ocspd>
+B<ocspd>
 [B<-d>]
 [B<-p n>]
 [B<-b address>]
@@ -21,9 +21,11 @@
 
 =head1 DESCRIPTION
 
-The B<openca-ocspd> is an RFC2560 compliant OCSPD responder. It can
-be used to verify the status of a certificate using OCSP clients
-(such as Mozilla/Netscape7).
+The B<ocspd> is an RFC2560 compliant Online Certificate Status Protocol (OCSP)
+responder. The responder is capable of answering complex OCSP requests and it
+can be used to verify the status of a certificate from OCSP clients. An example
+of a configuration file and a way to start and make requests to the responder
+can be found in the /etc/ocspd directory.
 
 =head1 OPTIONS
 
@@ -45,7 +47,7 @@
 =item B<-c file>
 
 specifies the configuration file to be loaded. Default file loaded
-is B</usr/local/etc/ocspd.conf>.
+is B</etc/ocspd/ocspd.xml>.
 
 =item B<-md digest>
 
@@ -94,7 +96,7 @@
 
 =over 4
 
-	openca-ocspd -c contrib/ocspd.conf 
+	ocspd -c /etc/ocspd/ocspd.xml
 
 =back
 
@@ -114,4 +116,4 @@
 
 =cut
 
-
+=back
